### PR TITLE
Handle err travis

### PR DESCRIPTION
**Pull request by phongt**
This pull request for 2 issues detected by ci [Build #87](https://travis-ci.org/github/UmlautSoftwareDevelopmentAccount/AP-CoreTypes/builds/702430535)

1. Improve the build command in travis.yml
2. fix a small issue of ara/core/array.h implementation catch by clang.

### CODE REVIEW

File: tools/build_ci.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+# Prints an error message in bold red then exits
+function die() {
+    red "${1}"
+    exit 1
+}
+
+function usage() {
+    echo "${0}" -c -h -a
+    echo "-c: Use clang compiler"
+    echo "-a: Enable address sanitizer"
```
Comment (bburda) [inline tools/build_ci.sh:26]: I wonder, maybe we should add Address Sanitizer, as a default option?

Comment (slci) [reply tools/build_ci.sh:26]: IMO, it should be kept as an option. But what about other sanitizers? Meson has options for `thread`, `undefined`, `memory` and `address,undefined`

**Comment by slci** [comment]
> Looks good, approved. But maybe someone else has something to add? Maybe @slci @insooth ?

I think, similar change should also go to https://github.com/UmlautSoftwareDevelopmentAccount/AP-Functional-Cluster-Template

**Comment by phongt** [comment]
@bburda  please take a look. I added 2 more changes 695de90 and 4b8dacf

**Pull request merged**